app/validations/address.js

```diff
@@ -0,0 +1,48 @@
+import { validatePresence } from 'ember-changeset-validations/validators';
+
+export function getAddressValidations(isAlwaysRequired = false) {
+  const REQUIRED_MESSAGE = 'Vul het volledige adres in';
+
+  return {
+    street: validatePresence({
+      presence: true,
+      ignoreBlank: true,
+      message: REQUIRED_MESSAGE,
+      on: isAlwaysRequired
+        ? null
+        : ['number', 'postcode', 'municipality', 'province'],
```

Comment:
If isAlwaysRequired = `false` then we the field is only required if one of the other address fields are filled in (which is what "on" does.

Reply:
I didn't understand the logic: then in that case if we didn't fill any of the fields, none is required?

Reply:
https://github.com/poteto/ember-changeset-validations#on-option-for-presence